--- src/notify.c
+++ src/notify.c
@@ -610,12 +610,18 @@
     if (strlen(actives[id].messages[0]) > MAX_BOX_MSG_LEN - 3)
         strcpy(actives[id].messages[0] + MAX_BOX_MSG_LEN - 3, "...");
 
+#ifdef NOTIFY_CHECK_VERSION
     actives[id].box = notify_notification_new(actives[id].title, actives[id].messages[0], NULL);
+#else
+    actives[id].box = notify_notification_new(actives[id].title, actives[id].messages[0], NULL, NULL);
+#endif
     actives[id].size++;
     actives[id].n_timeout = get_unix_time() + Control.notif_timeout / 1000;
 
     notify_notification_set_timeout(actives[id].box, Control.notif_timeout);
+#ifdef NOTIFY_CHECK_VERSION
     notify_notification_set_app_name(actives[id].box, "toxic");
+#endif
     /*notify_notification_add_action(actives[id].box, "lel", "default", m_notify_action, self, NULL);*/
     notify_notification_show(actives[id].box, NULL);
 
@@ -710,12 +716,18 @@
         strcpy(actives[id].messages[0] + MAX_BOX_MSG_LEN - 3, "...");
 
     actives[id].active = 1;
+#ifdef NOTIFY_CHECK_VERSION
     actives[id].box = notify_notification_new(actives[id].title, actives[id].messages[0], NULL);
+#else
+    actives[id].box = notify_notification_new(actives[id].title, actives[id].messages[0], NULL, NULL);
+#endif
     actives[id].size ++;
     actives[id].n_timeout = get_unix_time() + Control.notif_timeout / 1000;
 
     notify_notification_set_timeout(actives[id].box, Control.notif_timeout);
+#ifdef NOTIFY_CHECK_VERSION
     notify_notification_set_app_name(actives[id].box, "toxic");
+#endif
     /*notify_notification_add_action(actives[id].box, "lel", "default", m_notify_action, self, NULL);*/
     notify_notification_show(actives[id].box, NULL);
 
