--- src/audio.c
+++ src/audio.c
@@ -442,15 +442,15 @@
 
         // TODO move this code to filter_audio.c
         #ifdef AUDIO_FILTERING
-            if (!f_a && audio_filtering_enabled) {
+            if (!f_a && settings.audiofilter_enabled) {
                 f_a = new_filter_audio(UTOX_DEFAULT_SAMPLE_RATE_A);
                 if (!f_a) {
-                    audio_filtering_enabled = 0;
+                    settings.audiofilter_enabled = 0;
                     debug("filter audio failed\n");
                 } else {
                     debug("filter audio on\n");
                 }
-            } else if (f_a && !audio_filtering_enabled) {
+            } else if (f_a && !settings.audiofilter_enabled) {
                 kill_filter_audio(f_a);
                 f_a = NULL;
                 debug("filter audio off\n");
