--- src/utox.desktop
+++ src/utox.desktop
@@ -3,11 +3,14 @@
 Type=Application
 Name=uTox
 Comment=A lightweight Tox client
-Comment[ru]=легковесный клиент Tox
+Comment[ru]=Легковесный клиент Tox
 Comment[de]=Ein leichtgewichtiger Tox Client
+GenericName=A lightweight Tox client
+GenericName[ru]=Легковесный клиент Tox
+GenericName[de]=Ein leichtgewichtiger Tox Client
 TryExec=utox
 Exec=utox
 Icon=utox
-Categories=InstantMessaging;AudioVideo;Network;
+Categories=Network;InstantMessaging;P2P;
 Terminal=false
 MimeType=x-scheme-handler/tox;
