--- Makefile
+++ Makefile
@@ -26,9 +26,9 @@
 
 UNAME_S := $(shell uname -s)
 
-CFLAGS += -g -Wall -Wshadow -pthread -std=gnu99
-CFLAGS += $(shell pkg-config --cflags $(DEPS))
-LDFLAGS = -pthread -lm
+CFLAGS   = -g -Wall -Wshadow -pthread -std=gnu99
+CFLAGS  += $(shell pkg-config --cflags $(DEPS))
+LDFLAGS  = -pthread -lm
 LDFLAGS += $(shell pkg-config --libs $(DEPS))
 
 ifneq ($(DBUS), 1)
@@ -51,6 +51,9 @@
     LDFLAGS += -lresolv -ldl
 endif
 
+CFLAGS  += $(USER_CFLAGS)
+LDFLAGS += $(USER_LDFLAGS)
+
 DESTDIR ?=
 PREFIX ?= /usr/local
 
@@ -62,7 +65,7 @@
 
 utox: $(OBJ)
 	@echo "  LD    $@"
-	@$(CC) $(CFLAGS) -o utox $(OBJ) $(LDFLAGS)
+	$(CC) $(CFLAGS) -o utox $(OBJ) $(LDFLAGS)
 
 install: utox
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
@@ -110,7 +113,7 @@
 
 .c.o:
 	@echo "  CC    $@"
-	@$(CC) $(CFLAGS) -o $@ -c -DGIT_VERSION=\"$(GIT_V)\" $<
+	$(CC) $(CFLAGS) -o $@ -c -DGIT_VERSION=\"$(GIT_V)\" $<
 
 clean:
 	rm -f utox *.o png/*.o
