--- Makefile
+++ Makefile
@@ -11,8 +11,8 @@
 UNAME_S := $(shell uname -s)
 UNAME_O := $(shell uname -o)
 
-CFLAGS += -g -Wall -Wshadow -pthread -std=gnu99 -fno-strict-aliasing
-LDFLAGS += -pthread -lm
+CFLAGS  = -g -Wall -Wshadow -pthread -std=gnu99 -fno-strict-aliasing
+LDFLAGS = -pthread -lm
 
 ifeq ($(FILTER_AUDIO), 1)
 	DEPS += filteraudio
@@ -78,6 +78,8 @@
 	TRAY_GEN = x86_64-w64-mingw32-windres icons/icon.rc -O coff -o
 endif
 
+CFLAGS  += $(USER_CFLAGS)
+LDFLAGS += $(USER_LDFLAGS)
 
 DESTDIR ?=
 PREFIX ?= /usr/local
@@ -92,7 +94,7 @@
 
 utox: $(OBJ) $(OS_OBJ) $(TRAY_OBJ)
 	@echo "  LD    $@"
-	@$(CC) $(CFLAGS) -o $(OUT_FILE) $(OBJ) $(OS_OBJ) $(TRAY_OBJ) $(LDFLAGS)
+	$(CC) $(CFLAGS) -o $(OUT_FILE) $(OBJ) $(OS_OBJ) $(TRAY_OBJ) $(LDFLAGS)
 
 install: utox
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
@@ -138,11 +140,11 @@
 
 $(OBJ): %.o: %.c $(HEADERS)
 	@echo "  CC    $@"
-	@$(CC) $(CFLAGS) -o $@ -c -DGIT_VERSION=\"$(GIT_V)\" $<
+	$(CC) $(CFLAGS) -o $@ -c -DGIT_VERSION=\"$(GIT_V)\" $<
 
 $(OS_OBJ): %.o: %.c $(HEADERS)
 	@echo "  CC    $@"
-	@$(CC) $(CFLAGS) -o $@ -c -DGIT_VERSION=\"$(GIT_V)\" $<
+	$(CC) $(CFLAGS) -o $@ -c -DGIT_VERSION=\"$(GIT_V)\" $<
 
 $(TRAY_OBJ):
 	$(TRAY_GEN) $(TRAY_OBJ)
