--- Makefile
+++ Makefile
@@ -9,8 +9,11 @@
 UNAME_S		:= $(shell uname -s)
 UNAME_O		:= $(shell uname -o)
 
-CFLAGS		+= -g -Wall -Wshadow -pthread -std=gnu99 -fno-strict-aliasing
-LDFLAGS		+= -pthread -lm
+CFLAGS		= -g -Wall -Wshadow -pthread -std=gnu99 -fno-strict-aliasing
+LDFLAGS		= -pthread -lm
+
+CFLAGS		+= $(USER_CFLAGS)
+LDFLAGS		+= $(USER_LDFLAGS)
 
 DESTDIR		?=
 PREFIX		?= /usr/local
@@ -57,6 +60,7 @@
 	endif
 
 	UNX_CFLAGS += $(shell $(PKG_CONFIG) --cflags $(DEPS))
+	CFLAGS  += $(UNX_CFLAGS)
 	LDFLAGS += -lresolv -ldl $(shell $(PKG_CONFIG) --libs $(DEPS))
 	TRAY_GEN = $(LD) -r -b binary icons/utox-128x128.png -o
 
@@ -176,7 +180,7 @@
 
 $(OBJ): %.o: %.c $(HEADERS)
 	@echo "      CC    $@"
-	@$(CC) $(CFLAGS) -c -o $@ $<
+	$(CC) $(CFLAGS) -c -o $@ $<
 
 # This will only work on Cygwin
 $(WIN_OBJ): %.o: %.c $(HEADERS)
@@ -185,11 +189,11 @@
 
 $(UNX_OBJ): %.o: %.c $(HEADERS)
 	@echo "  UNX_CC    $@"
-	@$(CC) $(CFLAGS) -o $@ -c $<
+	$(CC) $(CFLAGS) -o $@ -c $<
 
 $(TRAY_OBJ):
 	@echo "Creating tray icon"
-	@$(TRAY_GEN) $@
+	$(TRAY_GEN) $@
 
 clean: win-clean
 	rm -f ./$(OUT_FILE) ./$(TRAY_OBJ) ./src/*.o ./src/xlib/*.o
