--- src/widget/systemtrayicon.h
+++ src/widget/systemtrayicon.h
@@ -38,6 +38,7 @@
     void hide();
     void setVisible(bool);
     void setIcon(QIcon &icon);
+    SystrayBackendType backend() const { return backendType; }
 
 signals:
     void activated(QSystemTrayIcon::ActivationReason);
--- src/widget/widget.cpp
+++ src/widget/widget.cpp
@@ -1801,7 +1801,13 @@
             trayMenu->addAction(actionQuit);
             icon->setContextMenu(trayMenu);
 
+            // don't activate qTox widget on tray icon click in Unity backend (see #3419)
+            #ifdef ENABLE_SYSTRAY_UNITY_BACKEND
+            if (icon->backend() != SystrayBackendType::Unity)
+                connect(icon, &SystemTrayIcon::activated, this, &Widget::onIconClick);
+            #else
             connect(icon, &SystemTrayIcon::activated, this, &Widget::onIconClick);
+            #endif
 
             if (Settings::getInstance().getShowSystemTray())
             {
