--- qtox.pro
+++ qtox.pro
@@ -17,7 +17,7 @@
 #    See the COPYING file for more details.
 
 
-QT       += core gui network xml opengl sql svg
+QT       += core gui network xml sql svg
 greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
 
 TARGET    = qtox
@@ -59,7 +59,6 @@
 # This hack returns 0 on batch (windows), but executes "date +%s" or return 0 if it fails on bash (linux/mac)
 TIMESTAMP = $$system($1 2>null||echo 0||a;rm null;date +%s||echo 0) # I'm so sorry
 DEFINES += TIMESTAMP=$$TIMESTAMP
-DEFINES += LOG_TO_FILE
 DEFINES += QT_MESSAGELOGCONTEXT
 
 contains(DISABLE_PLATFORM_EXT, YES) {
@@ -159,8 +158,8 @@
             LIBS += -Wl,-Bdynamic -lv4l1 -lv4l2 -lavformat -lavcodec -lavutil -lswscale -lusb-1.0
             LIBS += -lqrencode -lsqlcipher
         } else {
-            LIBS += -L$$PWD/libs/lib/ -ltoxcore -ltoxav -ltoxencryptsave -ltoxdns -lvpx -lsodium -lopenal -lavformat -lavdevice -lavcodec -lavutil -lswscale
-            LIBS += -lqrencode -lsqlcipher
+            LIBS += -L$$PWD/libs/lib/ -ltoxcore -ltoxav -ltoxencryptsave -ltoxdns -lvpx -lsodium -lopenal -lavdevice -lavfilter -lswscale -lpostproc -lavformat -lavcodec -lswresample -lavutil -lopus -lz -lbz2
+            LIBS += -lqrencode -lsqlcipher -lcrypto -ldl
         }
 
         contains(DEFINES, QTOX_PLATFORM_EXT) {
@@ -192,8 +191,6 @@
 
         PKGCONFIG += glib-2.0 gtk+-2.0 atk
         PKGCONFIG += cairo gdk-pixbuf-2.0 pango
-        PKGCONFIG += libavformat libavdevice libavcodec
-        PKGCONFIG += libavutil libswscale
         PKGCONFIG += appindicator-0.1 dbusmenu-glib-0.4
     }
 
@@ -203,8 +200,6 @@
 
         PKGCONFIG += glib-2.0 gtk+-2.0 atk
         PKGCONFIG += cairo gdk-pixbuf-2.0 pango
-        PKGCONFIG += libavformat libavdevice libavcodec
-        PKGCONFIG += libavutil libswscale
 
         SOURCES +=     src/platform/statusnotifier/closures.c \
         src/platform/statusnotifier/enums.c \
